index.html

```diff
@@ -2,80 +2,1625 @@
+
+    </div>
+  </section>
+
+  <!-- End Of Sports  -->
+
+  <!------------Subscribe------------->
+
+  <section class="subscribe">
+    <div id="subnews-block" class="position-relative border-bottom">
+      <div class="row">
+        <div class="col-lg-8">
+          <div class="sb-title text-uppercase">subscribe</div>
+          <div
+            class="subs-block d-flex flex-column align-items-center justify-content-center position-relative flex-md-row">
+            <div class="subs-image">
+              <div class="mag-thumb prev">
+                <img
+                  src="https://d.newsweek.com/en/full/1516486/cover-august-09-2019.jpg?w=170&q=95&f=23cfe68d3e6aaeba28a49957c89b1d3e"
+                  alt="">
+              </div>
+              <div class="mag-thumb current">
+                <img
+                  src="https://d.newsweek.com/en/full/1518227/cover-august-16-23-2019.jpg?w=170&q=95&f=350d013b3af6e3980dbaccd062943428"
+                  alt="">
+              </div>
+            </div>
+            <div class="subs-info pl-4">
+              <h3>In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h3>
+              <p>
+                All Access + Weekly Delivery. <br>
+                Choose A Membership That's Perfect for You!
+              </p>
+              <a href="#" class="text-uppercase text-white">Subscribe</a>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-4 ">
+          <div class="sb-title text-uppercase">Newsletter</div>
+          <div class="nletter-block d-flex flex-column flex-md-row">
+            <div class="nletter-image position relative"
+              style="background: url('https://g.newsweek.com/themes/newsweek/images/subscribe-img2.gif') center center / cover no-repeat;">
+            </div>
+            <div class="nletter-info h-100 w-100">
+              <h3 class="mb-3">Daily Briefing</h3>
+              <p class="mb-4">Take back your inbox.
+                Sign up for our newsletter and start making sense of the world around you.</p>
+              <a href="#" class="text-uppercase text-white">Sign up</a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+ </div>
+</div>
+</main>
+<!-- End Of Subscribe -->
+
+<!-- Footer -->
+
+<footer>
+    <div class="footer-top">
+      <div class="row">
+        <div class="col-md-9 logo-footer">
+          <a href="#"><img src="images/footer-logo-red.png" alt="logopix" class="img"></a>
+          <span class="copyright">&copy 2019 Newsweek</span>
+        </div>
+
+        <div class="col-md-3 media-icons">
+          <a href="#"><i class="fab fa-facebook-square"></i></a>
+          <a href="#"><i class="fab fa-twitter"></i></a>
+          <a href="#"><i class="fab fa-instagram"></i></a>
+          <a href="#"><i class="fab fa-tumblr"></i></a>
+          <a href="#"><i class="fab fa-linkedin-in"></i></a>
+        </div>
+      </div>
+    </div>
```

Comment:
#### Footer

- [ ] Kindly fix this part. It really is different than the original in all screen sizes.

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/88110132-0a775080-cb82-11ea-8869-35da70a97c9e.png)  | ![image](https://user-images.githubusercontent.com/61479648/88110192-25e25b80-cb82-11ea-8718-ee032381b62d.png) |